packages/server/modules/blobstorage/objectStorage.ts

```diff
@@ -51,32 +61,45 @@ const getObjectStorage = () => ({
   createBucket: createS3Bucket()
 })
 
-const sendCommand = async (command) => {
+const sendCommand = async <CommandOutput>(
+  command: (Bucket: string) => Command<any, CommandOutput, any, any, any>
+) => {
   const { client, Bucket } = getObjectStorage()
   try {
-    return await client.send(command(Bucket))
+    const ret = (await client.send(
+      command(Bucket) as Command<any, any, any, any, any>
+    )) as CommandOutput
```

Comment:
It's already `as CommandOutput` in practice, the type of CommandOutput is auto inferred from the actual command you pass in. The only reason I need to do this is because of all of the abstraction and generics behind the scene which break something - I dunno, it wasn't obvious to me how to avoid it.